shell/common/animator.cc

```diff
@@ -168,6 +168,9 @@ void Animator::DrawLastLayerTree() {
 }
 
 void Animator::RequestFrame(bool regenerate_layer_tree) {
+  if (delegate_.AnimatorShouldPause()) {
```

Comment:
Given that this is actually stopping and theres no equivalent of resuming from a paused state, I think we want to call this `AnimatorStopRequested()`.